pymc/distributions/timeseries.py

```diff
@@ -214,27 +215,14 @@ def logp(
         init: at.Variable,
         steps: at.Variable,
     ) -> at.TensorVariable:
-        """Calculate log-probability of Gaussian Random Walk distribution at specified value.
-
-        Parameters
-        ----------
-        value: at.Variable,
-        mu: at.Variable,
-        sigma: at.Variable,
-        init: at.Variable, Not used
-        steps: at.Variable, Not used
```

Comment:
These do not follow the style used in our docs and are wrong, because init and steps are both used. In any case, the logp function should be hidden from the users...

Reply:
Even if its hidden from users we should write docstrings for developers. This one isnt all that useful so deleting is fine. Just want to note developers are a form of users too!